1001-2000/LIT1812GospelLuke.xml

```diff
@@ -43,101 +43,145 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+         <change who="ABr" when="2020-12-04">Added type:="colophon" to xml:id="PostscriptLuke" and label Colophon of the Gospel of Luke</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
         <div type="edition">
+          <listBibl type="secondary">
+             <bibl>
+                <ptr target="bm:Dillmann1848Catalogus"/>
+                <citedRange unit="page">15</citedRange>
+             </bibl>
+             <bibl>
+                <ptr target="bm:Nicoll1821Bodleianae"/>
+                <citedRange unit="page">17-18</citedRange>
+             </bibl>
+          </listBibl>
+          <note> The incipits and the explicits are taken from <ref type="mss" corresp="GCA001"/>
+           </note>
 
-           <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="BiographyLuke">
-               <label>Introduction and Biography of Luke</label>
-              <listBibl type="secondary">
-                 <bibl>
-                    <ptr target="bm:Dillmann1848Catalogus"/>
-                    <citedRange unit="page">15</citedRange>
-                 </bibl>
-                 <bibl>
-                    <ptr target="bm:Nicoll1821Bodleianae"/>
-                    <citedRange unit="page">17-18</citedRange>
-                 </bibl>
-              </listBibl>
+        <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="LukeGospelPreface">
+          <label>Preface to the Gospel of Luke</label>
+           <div type="textpart" subtype="subtextunit" xml:lang="gez" xml:id="SupplicationPrefaceLuke">
+              <label>Supplication in the Preface to the Gospel of Luke</label>
+              <div type="textpart" subtype="incipit"><ab>በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩፡ አምላክ። ዘመሓረነ፡ በረድኤቱ፡
+               እምድኅረ፡ ስሕተት፡ ወዑረት። ወኣርኣየነ፡ ተመርሖ፡ እመድኅረ፡ እከይ፡ ወተኃጕሎት፡</ab></div>
+              <div type="textpart" subtype="explicit"><ab>ወስብሓት፡ ሎቱ፡ ዕዘዝ፡ አልቦ፡ አምላክ፡ ዘእንበሌሁ።</ab></div>
             </div>
-
-            <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="TituliLuke">
-               <label>Tituli of the Gospel of Luke</label>
+            <div type="textpart" subtype="title expanded" xml:id="TitlePrefaceLuke"><ab>ንቀድም፡ በረድኤተ፡
+             እግዚአብሔር፡ በጽሒፈ፡ መቅድመ፡ ሉቃስ፡ ኅሩይ፡ ፩፡ እም፸ወ፪፡ አርድእት፡ ኅሩያን።</ab></div>
+            <div type="textpart" subtype="subtextunit" xml:lang="gez" xml:id="BiographyLuke">
+              <label>Biography of Luke</label>
+              <div type="textpart" subtype="incipit"><ab>ዝንቱሰ፡ ሉቃስ፡ ነበረ፡ ዓቃቤ፡ ሥራይ፡ በኣንጾኪያ። ወኮነ፡ ረድኦ፡
+               ለእግዚእነ። ወውእቱ፡ ቀለዮጳ፡ ዘተርእዮ፡ እግዚእነ፡ በፍኖተ፡ ኢማሑስ፡ በዕለተ፡ ትንሣኤ።</ab></div>
+              <div type="textpart" subtype="explicit"><ab>ወኣዕረፈ፡ ከዊኖ፡ ሰማዕተ፡ በሮሜ፡ አመ፡ ፳ወ፪፡ ለጥቅምት።</ab></div>
+            </div>
+            <div type="textpart" subtype="subtextunit" xml:lang="gez" xml:id="EusebianSectionsLuke">
+              <label>Statement about the Eusebian Sections in the Gospel of Luke</label>
```

Comment:
In the previous records I read "Statement about Eusebian Sections", with no article. I would conform all labels in one formula with or without article. I think the article is needed, but someone more trained than me might confirm this.

Reply:
Sure, with the article. Thank you!